src/link_manager/LinkManagerStateMachineActiveStandby.cpp

```diff
@@ -829,6 +832,8 @@ void ActiveStandbyStateMachine::handleSuspendTimerExpiry()
         CompositeState currState = mCompositeState;
         enterMuxWaitState(mCompositeState);
         LOGINFO_MUX_STATE_TRANSITION(mMuxPortConfig.getPortName(), currState, mCompositeState);
+        // detect if link/server recovers
+        mDetectLinkFnPtr();
```

Comment:
Let's send the heartbeats first, then `enterMuxWaitState` (probe mux state).

Reply:
Also is there a way to make sure all 3 `sendHeartbeat` are dispatched and executed before probing mux state.

Reply:
Thanks, force the sending in link detect.